src/DocsParser.ts

```diff
@@ -146,6 +146,16 @@ export class DocsParser {
           'HTMLElement documentation should not be considered a class',
         );
       }
+      let electronProcess = { main: true, renderer: true };
+      for (let tk of tokens) {
+        if (tk.type == 'inline' && tk.content.indexOf('Process') != -1) {
+          electronProcess = {
+            main: tk.content.indexOf('[Main]') != -1,
```

Comment:
Worth noting I think this is _kinda_ hacky, it's parsing the raw string instead of going token by token and for instance ensuring that this is a link.